warehouse/ingest-configuration/src/main/resources/config/edge-definitions.xml

```diff
@@ -35,6 +35,11 @@ http://www.springframework.org/schema/util/spring-util-4.0.xsd">
         <property name="edgeAttribute2" value="NAME"/>
         <property name="edgeAttribute3" value="ID"/>
         <property name="activityDateField" value = "PREMIERED"/>
+        <property name="allowedTypeCategories">
```

Comment:
Do we want separate allowed type categories for the source and the sink ?

Reply:
I would let the bidirectional config dictate that.